docs/advanced_topics/privacy.rst

```diff
@@ -3,9 +3,33 @@
 Private pages
 =============
 
-Users with publish permission on a page can set it to be private by clicking the 'Privacy' control in the top right corner of the page explorer or editing interface, and setting a password. Users visiting this page, or any of its subpages, will be prompted to enter a password before they can view the page.
+Users with publish permission on a page can set it to be private by clicking the 'Privacy' control in the top right corner of the page explorer or editing interface. This sets a restriction on who is allowed to view the page and its sub-pages. Several different kinds of restriction are available:
 
-Private pages work on Wagtail out of the box - the site implementer does not need to do anything to set them up. However, the default "password required" form is only a bare-bones HTML page, and site implementers may wish to replace this with a page customised to their site design.
+ * **Visible if logged in:** The user must log in to view the page. All user accounts are granted access, regardless of permission level.
+ * **Accessible with the following password:** The user must enter the given password to view the page. This is appropriate for situations where you want to share a page with a trusted group of people, but giving them individual user accounts would be overkill. The same password is shared between all users, and this works independently of any user accounts that exist on the site.
```

Comment:
"Accessible" is the wording [used on the existing label](https://github.com/torchbox/wagtail/blob/84c36c1ec2eca8fd51028e7c46a6046ba64c8dbd/wagtail/wagtailadmin/forms.py#L185) - the two "visible" options are the new ones.

I'm torn about which word is better to standardise upon - "visible" sounds better, but "accessible" is probably more technically correct, because a protected page can still be "visible" in the sense that it appears in menus (unless the developer has specifically applied a `public()` filter on the queryset). "Viewable"? Perhaps I'm overthinking this :-)